src/coreclr/scripts/superpmi.py

```diff
@@ -1285,6 +1285,8 @@ def print_superpmi_failure_code(return_code, coreclr_args):
         logging.warning("Compilation failures")
     elif return_code == 2:
         logging.warning("Asm diffs found")
+    elif return_code == 3:
+        logging.warning("SPMI misses found")
```

Comment:
Can you please spell this out, e.g., "SuperPMI missing data encountered".

Reply:
done